projects/angular/src/accordion/accordion-panel.spec.ts

```diff
@@ -259,6 +259,11 @@ describe('ClrAccordionPanel', () => {
       expect(panelGroup.classList.contains('clr-accordion-panel-open')).toBe(true);
     });
 
+    it('should not have the [role]="modal" attribute', () => {
```

Comment:
the test name says role=modal but the test is querying for role=group?

Reply:
Thank. I had modals on the mind. I wIll update.